spacetime/tests/test_linear_scalar.py

```diff
@@ -133,4 +134,100 @@ def _march():
             self.assertEqual(self.svr.get_so0(0).ndarray.tolist(),
                              svr2.get_so0(0).ndarray.tolist())
 
+
+class LinearScalarGridTestTC(unittest.TestCase):
+    """
+    Compare linear solver's solution wtih exact solution, because solutions of computation field is a vector,
```

Comment:
Per the 79-char suggestion of PEP-8, lines in the docstrings seem to be too long.

Reply:
I modified docstring that make this docstring fit PEP-8 style in https://github.com/yungyuc/turgon/pull/32/commits/e9915d61ee27a54507790dc562574b80dd0fdcfc